node/bft/src/primary.rs

```diff
@@ -1297,15 +1301,21 @@ impl<N: Network> Primary<N> {
             bail!("Round {batch_round} is too far in the past")
         }
 
+        // TODO: should we get the current or previous committee?
```

Comment:
I think the current committee is likely best? After all this represents the latest valid state update the node is aware of?

Reply:
Yeah agreed, let's see what the test suite thinks